Plugin.php

```diff
@@ -20,71 +22,159 @@ public function pluginDetails()
+                        'icon'  => 'icon-database',
+                        'url'   => Backend::url('october/test/users'),
                     ],
                     'countries' => [
-                        'label'       => 'Countries',
-                        'icon'        => 'icon-database',
-                        'url'         => Backend::url('october/test/countries'),
+                        'label' => 'Countries',
+                        'icon'  => 'icon-database',
+                        'url'   => Backend::url('october/test/countries'),
                     ],
-                    'reviews' => [
-                        'label'       => 'Reviews',
-                        'icon'        => 'icon-database',
-                        'url'         => Backend::url('october/test/reviews'),
+                    'reviews'   => [
+                        'label' => 'Reviews',
+                        'icon'  => 'icon-database',
+                        'url'   => Backend::url('october/test/reviews'),
                     ],
                     'galleries' => [
-                        'label'       => 'Galleries',
-                        'icon'        => 'icon-database',
-                        'url'         => Backend::url('october/test/galleries'),
+                        'label' => 'Galleries',
+                        'icon'  => 'icon-database',
+                        'url'   => Backend::url('october/test/galleries'),
+                    ],
+                    'trees'     => [
+                        'label' => 'Trees',
+                        'icon'  => 'icon-database',
+                        'url'   => Backend::url('october/test/trees'),
                     ],
-                    'trees' => [
-                        'label'       => 'Trees',
-                        'icon'        => 'icon-database',
-                        'url'         => Backend::url('october/test/trees'),
+                    'pages'     => [
+                        'label' => 'Pages',
+                        'icon'  => 'icon-database',
+                        'url'   => Backend::url('october/test/pages'),
                     ],
-                    'pages' => [
-                        'label'       => 'Pages',
-                        'icon'        => 'icon-database',
-                        'url'         => Backend::url('october/test/pages'),
-                    ]
-                ]
-            ]
+                ],
+            ],
         ];
     }
 
     public function registerFormWidgets()
     {
         return [
             'October\Test\FormWidgets\TimeChecker' => [
-                'code'  => 'timecheckertest'
-            ]
+                'code' => 'timecheckertest',
+            ],
         ];
     }
+
+    public function boot()
+    {
+        if (\App::runningInBackend()) {
+            $pluginManager = \System\Classes\PluginManager::instance();
+            if ($pluginManager->hasPlugin('RainLab.Pages')) {
```

Comment:
Strictly speaking, the above three lines are unnecessary since the `backend.form.extendFields` event will only run on the backend and the `instanceof` check prevents it from affecting anything but the targetted form.

Reply:
Excellent point - thanks for the heads up!